nebula-query-and-search/main/classes/AggregateQuery.cls

```diff
@@ -106,6 +136,11 @@ global class AggregateQuery extends SOQL {
     return this.setHasChanged();
   }
 
+  global AggregateQuery withAccessLevel(System.AccessLevel mode) {
+    super.doWithAccessLevel(mode);
```

Comment:
For consistency, I think the variable should be called `accessLevel` instead of `mode`

```suggestion
  global AggregateQuery withAccessLevel(System.AccessLevel accessLevel) {
    super.doWithAccessLevel(accessLevel);
```